api/v1/constants.go

```diff
@@ -1,13 +1,17 @@
 package v1
 
 const (
-	SyncCanaryJobsSchedule             = "@every 2m"
-	SyncSystemsJobsSchedule            = "@every 5m"
-	ComponentRunSchedule               = "@every 2m"
-	ComponentStatusSummarySyncSchedule = "@every 1m"
-	ComponentCheckSchedule             = "@every 2m"
-	ComponentConfigSchedule            = "@every 2m"
-	ComponentCostSchedule              = "@every 1h"
-	CheckStatusSummarySchedule         = "@every 1m"
-	CheckStatusDeleteSchedule          = "@every 24h"
+	SyncCanaryJobsSchedule                  = "@every 2m"
+	SyncSystemsJobsSchedule                 = "@every 5m"
+	ComponentRunSchedule                    = "@every 2m"
+	ComponentStatusSummarySyncSchedule      = "@every 1m"
+	ComponentCheckSchedule                  = "@every 2m"
+	ComponentConfigSchedule                 = "@every 2m"
+	ComponentCostSchedule                   = "@every 1h"
+	CheckStatusSummarySchedule              = "@every 1m"
+	CheckStatusesAggregate1hSchedule        = "@every 1h"
+	CheckStatusesAggregate1dSchedule        = "@every 24h"
+	CheckStatuses1hAggregatedDeleteSchedule = "@every 24h"
+	CheckStatuses1dAggregatedDeleteSchedule = "@every 24h"
+	CheckStatusDeleteSchedule               = "@every 24h"
```

Comment:
I would merge these 3 into a single job and schedule

Reply:
That's how I did it initially but these are 3 separate elementary operations. I'm not sure how to mark the status of the job if they fail partially.

Reply:
You can have different job names, all with the same schedule

Reply:
😁 that was quite silly of me.